includes/admin/views/html-order-shipment-summary.php

```diff
@@ -68,5 +65,28 @@
         $labelDescription
     );
 }
-
 echo '</ul>';
+
+echo "<hr>";
+
+/**
+ * Do show the Track & Trace status for all shipments.
+ */
+foreach ($shipments as $shipment_id => $shipment) {
+    $trackTrace = Arr::get($shipment, "track_trace");
+
+    /**
+     * Show Track & Trace status.
+     */
+    if (! $trackTrace) {
```

Comment:
Als er geen $trackTrace is, wil je waarschijnlijk wel aangeven dat het een concept is. Is dit datgene waar je ook een print knop naast hebt? Dan wil je ook een concept uit kunnen printen.

Reply:
Dit is er pas na het exporteren, het i'tje naast shipment options. Eigenlijk moet trackTrace dan  voor alle items ook bestaan, maar dit werd eerst ook (ongeveer) zo gecontroleerd dus dat heb ik behouden.